python/openassetio_traitgen/__main__.py

```diff
@@ -86,6 +83,14 @@ def _create_argparser():
         " directory if it does not exist",
     )
 
+    cmdline.add_argument(
```

Comment:
[fix] We should make this a required argument, or set a default.

Reply:
Yeah good spot, this was a stupid miss.

efdc4e8](https://github.com/OpenAssetIO/OpenAssetIO-TraitGen/pull/7/commits/efdc4e86e9b433d6150a638e02e3fb6757d74eb3)